cla_backend/apps/call_centre/management/commands/cla_superuser_organisation_perms.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management import BaseCommand
+from django.contrib.auth.models import Group, Permission
+
+
+class Command(BaseCommand):
+    help = "Assign CLA Superusers 'Can add organisation' and 'Can change organisation' permissions"
+
+    def handle(self, *args, **options):
+        group = Group.objects.get(name="CLA Superusers")
+        if not group:
```

Comment:
This `.get()` will throw a `Group.DoesNotExist` exception (which is the specific subclass of `ObjectDoesNotExist`) rather than returning something falsey, so you should try/except that

(or use a `filter()` first, check `.exists()` on the queryset that that returns, and then use a `.get()` with no arguments to go from the queryset to the object)